.github/workflows/blossom-ci.yml

```diff
@@ -24,8 +24,8 @@ jobs:
 
     # This job only runs for pull request comments
     if: |
-         contains( 'xwang233,jjsjann123,', format('{0},', github.actor)) &&
-         github.event.comment.body == '!dontusethis'

```

Comment:
cc'ing @xwang233 are we still using this one? I thought it's just a toy example.

Can we remove it now?

Reply:
will discuss this offline